sql-statements/sql-statement-with.md

```diff
@@ -82,9 +82,9 @@ WITH RECURSIVE cte(a) AS (SELECT 1 UNION SELECT a+1 FROM cte WHERE a < 5) SELECT
 
 ## MySQL compatibility
 
-* In strict mode, when the data length recursively calculated exceeds the data length of the seed part, TiDB will return a warning while MySQL will return an error. In non-strict mode, the behavior of TiDB is consistent with that of MySQL.
+* In the strict mode, when the data length recursively calculated exceeds the data length of the seed part, TiDB returns a warning while MySQL returns an error. In the non-strict mode, the behavior of TiDB is consistent with that of MySQL.
```

Comment:
@TomShawn MySQL refers to it as "strict mode" not "the strict mode". See: https://dev.mysql.com/doc/refman/8.0/en/sql-mode.html#sql-mode-strict for example.

Reply:
Removed in https://github.com/pingcap/docs/pull/5770/commits/9a1cf22b268b3865bf427c301d491878531fbab5. Thanks for letting me know.